index.php

```diff
@@ -19,7 +20,31 @@
             <h4 class="font-style-micro font-size-small margin-bottom-small text-align-center"><?php the_time('d F Y'); ?></h4>
             <h3 class="margin-bottom-small text-align-center"><?php the_title(); ?></h3>
 
-            <?php the_excerpt(); ?>
+    <?php
```

Comment:
why can't all this code here be in a function? Is basically has to be as this is to be implemented on other templates.